clang/lib/CodeGen/CodeGenModule.h

```diff
@@ -1622,6 +1622,9 @@ class CodeGenModule : public CodeGenTypeCache {
   void CreateFunctionTypeMetadataForIcall(const FunctionDecl *FD,
                                           llvm::Function *F);
 
+  /// Create and attach type metadata to the given call.
+  void CreateCalleeTypeMetadataForIcall(const QualType &QT, llvm::CallBase *CB);
```

Comment:
```suggestion
  void createCalleeTypeMetadataForIcall(const QualType &QT, llvm::CallBase *CB);
```
The existing API should probably be renamed as well, but that should happen separately.

Reply:
The existing API is named appropriately `CreateFunctionTypeMetadataForIcall` -- adds a `type` metadata to the Function as opposed to the new API `createCalleeTypeMetadataForIcall` which adds `callee_type` metadata to the indirect call instruction.

Reply:
Ah the capitalization! nevermind the previous comment. I'll fix the API names.